modules/monitoring-setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects.adoc

```diff
@@ -34,7 +40,7 @@ endif::openshift-dedicated,openshift-rosa[]
 $ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
 ----
 
-. Add the `enforcedSampleLimit` configuration to `data/config.yaml` to limit the number of samples that can be accepted per target scrape in user-defined projects:
+. Add the enforced limit and time interval configurations to `data/config.yaml`:
```

Comment:
I would recommend putting the configuration names back in this step because you are assuming the user knows what those are.

Reply:
But then we would have to list 6 parameters, which would make the sentence really long and complicated. Plus we explain everything in callout so no information is lost.